guides/09_other-tutorials/deploying-gradio-with-docker.md

```diff
@@ -0,0 +1,83 @@
+# Deploying a Gradio app with Docker
+
+Tags: DEPLOYMENT, DOCKER
+
+
+### Introduction
+
+Gradio is a powerful and intuitive Python library designed for creating web apps that showcase machine learning models. These web apps can be run locally, or [deployed on Hugging Face Spaces ](https://huggingface.co/spaces)for free. Or, you can deploy them on your servers in Docker containers. Dockerizing Gradio apps offers several benefits:
+
+- **Consistency**: Docker ensures that your Gradio app runs the same way, irrespective of where it is deployed, by packaging the application and its environment together.
+- **Portability**: Containers can be easily moved across different systems or cloud environments.
+- **Scalability**: Docker works well with orchestration systems like Kubernetes, allowing your app to scale up or down based on demand.
+
+## How to Dockerize a Gradio App
+
+Let's go through a simple example to understand how to containerize a Gradio app using Docker.
+
+#### Step 1: Create Your Gradio App
+
+First, we need a simple Gradio app. Let's create a Python file named `app.py` with the following content:
+
+```python
+import gradio as gr
+
+def greet(name):
+    return f"Hello {name}!"
+
+iface = gr.Interface(fn=greet, inputs="text", outputs="text").launch()
+```
+
+This app creates a simple interface that greets the user by name.
+
+#### Step 2: Create a Dockerfile
+
+Next, we'll create a Dockerfile to specify how our app should be built and run in a Docker container. Create a file named `Dockerfile` in the same directory as your app with the following content:
+
+```dockerfile
```

Comment:
Gonna add support for dockerfile syntax highlighting, right now this isn't rendering correctly